sigopt_sklearn/search.py

```diff
@@ -227,7 +228,19 @@ def _check_bounds():
         # generate sigopt experiment parameters
         return [_transform_param(name, bounds) for (name, bounds) in param_domains.items()]
 
-    def _create_sigopt_exp(self, conn, folds):
+    def create_experiment(self, num_folds):
```

Comment:
Yeah, I tried reusing `cv` here, but it gets messy. `cv` is only really instantiated once `check_cv` is called, and doing that requires `X`, `y`, and the estimator. We only have `X` and `y` when `fit` is called.